src/@types/event.ts

```diff
@@ -373,7 +373,7 @@ export interface AccountDataEvents extends SecretStorageAccountDataEvents {
     [EventType.PushRules]: IPushRules;
     [EventType.Direct]: { [userId: string]: string[] };
     [EventType.IgnoredUserList]: { [userId: string]: {} };
-    "m.secret_storage.default_key": { key: string };
+    "m.secret_storage.default_key": { key: string } | {};
```

Comment:
` | {}` means accept anything. I suggest reading https://www.totaltypescript.com/the-empty-object-type-in-typescript - also not sure if this is even wanted here as `setAccountData` already has `| Record<string, never>` to allow "deleting" account data events

Reply:
Oh TIL, thanks!

The rust sdk is setting an empty object and I used the same approach for constituency between our sdks

Reply:
Yep, but we still need to modify the `AccountDataClient` interface of `secret-storage` to able to do it here. (will do)